keps/NNNN-kep-template/README.md

```diff
@@ -332,6 +342,151 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+## Production Readiness Review Questionnaire
```

Comment:
I opened https://github.com/kubernetes/community/pull/4671 to update and slightly extend PRR questionaire. Please unify it before merging (assuming that PR will be merged by then).